webapp/javascript/redux/reducers/continuous.ts

```diff
@@ -337,7 +337,7 @@ export const continuousSlice = createSlice({
     /*******************************/
 
     builder.addCase(fetchTagExplorerView.pending, (state) => {
-      switch (state.diffView.type) {
```

Comment:
Is there any purpose to check loading state of diffView store section for tagExplorerView one? @eh-am

Reply:
most likely a typo